addroom.lic

```diff
@@ -17,24 +17,36 @@
   You can turn on debug mode by passing 'debug' as an argument with ;addroom debug
 =end
 
-$debug_mode = UserVars.add_room_debug || variable.include?('debug')
-$skip_checkout = variable.include?('manual')
+
+
+arg_definitions = [
+  [
+    {name: 'path', display:'path to map', regex:/^[\w\s\,]$/, variable:true, description: 'comma separated list of directions to map, use double quotes to wrap. "north, south"'}
+    {name: 'debug', regex:/debug/i, optional:true},
+    {name: 'manual', regex:/manual/i, optional:true, description:'Don\'t automatically checkout and commit mapdb to lich'},
```

Comment:
If it's not clear they probably should be using the script honestly. It's an optional toggle to prevent the script from checking out and committing the map db.